Echo.json5

```diff
@@ -62,7 +62,7 @@
     { name: 'AREnableMyCollectionInsightsPhase1Part1', value: true },
     { name: 'AREnableMyCollectionInsightsPhase1Part2', value: true },
     { name: 'AREnableMedianPriceChartCareerHighlights', value: true },
-    { name: 'AREnableNewImage', value: false },
+    { name: 'AREnableNewImage', value: true },
```

Comment:
**suggestion**: remove [dev label](https://github.com/artsy/eigen/blob/9c5604c95ff993ce26acbd07ccd23c19978ed338/src/palette/elements/OpaqueImageView/OpaqueImageView2.tsx#L185-L191) for staging environment

Reply:
🤔 maybe I could also introduce a dev toggle for this though!

Reply:
Yes, a good idea!